lib/lanpartyseating/logic/tournaments_logic.ex

```
@@ -52,10 +55,11 @@ defmodule Lanpartyseating.TournamentsLogic do
           {Lanpartyseating.Tasks.ExpireTournament, {tournament.end_date, tournament.id}}
         )
 
+        {:ok, tournaments} = get_upcoming_tournaments()
```

The `Repo.insert`'s error is not handled in this function either